.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,10 @@
 <!--
 Work-in-progress PRs are welcome as a way to get early feedback - just prefix
 the title with [WIP].
+
+Add the change in the changelog (except for test changes and docs updates).
+Please edit CHANGELOG.md and add the change under the appropriate category (NEW
```

Comment:
the changelog currently only has IMPROVEMENTS, not all three of these. seems to me like KISS would dictate that we just have IMPROVEMENTS and BUG FIXES as sections, but if you're modeling this after some other changelog that seems particularly effective, i'm happy to defer.

Reply:
I think we would have new features when the new `status` is implemented. So, we can use NEW FEATURES then. Let's keep it 🙂

Reply:
what rule would you apply to distinguish between an improvement and a feature?

Reply:
NEW FEATURE = new subcommands, flags, importer support
IMPROVEMENTS = enhancement on existing features and behavior

I see NEW FEATURE as a special section that would be at the top, which could be used to highlight important changes that affect the users. This could be a new way to do something,  support for something new or a breaking change to some existing behavior. And IMPROVEMENTS for the rest of the things, that a normal user need not care much.

Reply:
i still think we'll have some difficulty cleanly applying that rule to cleanly delineate between the two, but as long as we're looking at "NEW FEATURE" as a special section that we use to highlight things, i think it's probably OK.